src/Pagination/PaginateDirective.php

```diff
@@ -110,10 +121,24 @@ public function manipulateFieldDefinition(DocumentAST &$documentAST, FieldDefini
     public function resolveField(FieldValue $fieldValue): FieldValue
     {
         $fieldValue->setResolver(function ($root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): Paginator {
+            if ($this->directiveHasArgument('resolver')) {
+                $builderResolver = $this->getResolverFromArgument('resolver');
+
+                $query = $builderResolver($root, $args, $context, $resolveInfo);
```

Comment:
```suggestion
                $paginator = $this->getResolverFromArgument('resolver')($root, $args, $context, $resolveInfo);
```

Reply:
The variable names you used are misleading.

Reply:
Copy paste error sorry :) 
Changed to $resolverResolver